packages/unmock-node/src/__tests__/index.test.ts

```diff
@@ -38,6 +54,7 @@ test("unmock end to end node", async () => {
             value: "Test",
           },
         ],
-  });
+      },
+    );
   expect(typeof vid).toBe("number");
-});
+}, 20000);
```

Comment:
Is this a timeout setting? I've updated the timeout setting in #8 - I think it's cleaner to make it consistent across test (and more verbose that way too), by setting it in `jest.config.js` - [increase jest timeout](https://github.com/unmock/unmock-js/commit/6cf55fd5b2fff8af6ac7a3a859efb3bc320d5478)

Reply:
great idea! done